docs/tutorials/uniq-factories/creating-uniq-factories/how-to-create-uniq-metadata.md

```diff
@@ -0,0 +1,438 @@
+
+### Output File Explained
+
+It is important to understand that the `upload.json` file <ins>is not part of the actual metadata</ins>, rather it is generated as a help for you to understand where you _should_ upload/host the generated metadata files.
+
+For our example collection, the `upload.json` file should look like this:
+
+```json
+{
+    "collectionName": "Angry Bananas",
+    "factory": {
+        "hash": "1d5a17f4de3940e1891a1c98e51c9bd1ea2a137a264e10d1e542aeae89e1a0fd",
+        "url": "https://developers.ultra.io/uniq-collections/AngryBananas/1d5a17f4de3940e1891a1c98e51c9bd1ea2a137a264e10d1e542aeae89e1a0fd.json"
+    },
+    "defaultToken": {
+        "hash": "104ee04db49432ee9373993596dac3e0d10bd775a7da93451f2d7bdea5e16f1b",
+        "url": "https://developers.ultra.io/uniq-collections/AngryBananas/{serial_number}.json"
+    },
+    "tokens": [
+        {
+            "serialNumber": "1",
+            "hash": "73b97f61f68ab12929b34c893f010df91f51d8096658b10fc1deb6274b57f9cf",
+            "url": "https://developers.ultra.io/uniq-collections/AngryBananas/1.json"
+        },
+        {
+            "serialNumber": "2",
+            "hash": "7ebbcf3462136f8b61234a39fce20343873c13ff2e67235d2277ba36eb1f3c45",
+            "url": "https://developers.ultra.io/uniq-collections/AngryBananas/2.json"
+        }
+    ],
+    "media": {
+        "generated_media/969f723aff5d40fefce3749cc3be9eb3b6932fe10af6655954f5d85e5313edd2.png": "https://developers.ultra.io/uniq-collections/AngryBananas/969f723aff5d40fefce3749cc3be9eb3b6932fe10af6655954f5d85e5313edd2.png",
+        "generated_media/6057d2edc51714503c5c45795d4db998ac740eba5fa98beee99ca4aafc3def61.png": "https://developers.ultra.io/uniq-collections/AngryBananas/6057d2edc51714503c5c45795d4db998ac740eba5fa98beee99ca4aafc3def61.png",
+        "generated_media/6db53e6353bf3a99aec0c73a884e0b4fa642d50868cf29895c614800306e3205.png": "https://developers.ultra.io/uniq-collections/AngryBananas/6db53e6353bf3a99aec0c73a884e0b4fa642d50868cf29895c614800306e3205.png",
+        "generated_media/8719cb0820013ed612ac2045b360b6b1ad5ab510bf9c6ab6ca8c7e09f820ea7c.png": "https://developers.ultra.io/uniq-collections/AngryBananas/8719cb0820013ed612ac2045b360b6b1ad5ab510bf9c6ab6ca8c7e09f820ea7c.png"
+    },
+    "environment": {
+        "env": "developerDocs",
+        "tokenUriTemplate": "{serial_number}",
+        "url": "https://developers.ultra.io/uniq-collections",
+        "toolVersion": "1.4.8"
+    }
+}
+```
+
+Based on above details, we have uploaded the metadata and media files to the specified URLs:
+
+-   Factory metadata: https://developers.ultra.io/uniq-collections/AngryBananas/1d5a17f4de3940e1891a1c98e51c9bd1ea2a137a264e10d1e542aeae89e1a0fd.json
+-   Default uniq metadata: https://developers.ultra.io/uniq-collections/AngryBananas/{serial_number}.json
```

Comment:
This one is not a valid URL. You need to replace `{serial_number}` with actual token number and then it will be identical to `Uniq #1 metadata` and `Uniq #2 metadata`.

I think this one is supposed to be `defaultToken.json` (like what metadata tool generates). And then need to replace the `{serial_number}.json` with `defaultToken.json`. It is probably a bad practice to have files with names with `{}` in them